tests/arp/conftest.py

```diff
@@ -50,9 +50,27 @@ def intfs_for_test(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_fro
         intf1 = ports_for_test[0]
         intf2 = ports_for_test[1]
     else:
-        # Select port index 0 & 1 two interfaces for testing
-        intf1 = ports[0]
-        intf2 = ports[1]
+        # Select first 2 ports that are admin 'up'
+        if duthost.is_multi_asic:
```

Comment:
Better to avoid using "is_multi_asic" in the test logic. Use 'sonic.py', or 'sonic_asic.py' to add new APIs if required.